src/client/pages/ResendConsentEmail.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
 
 export const ResendConsentEmail = ({ token, queryParams }: Props) => {
 	return (
-		<MainLayout pageHeader="Link expired">
+		<MinimalLayout pageHeader="Link expired">
 			<MainBodyText>This link has expired.</MainBodyText>
 			<MainBodyText>To receive a new link, please click below.</MainBodyText>
```

Comment:
leadText?

Reply:
It looked weird to put both lines in `leadText` and felt wrong in the code to put only one of them in, but maybe yeah, the first line only.